src/mongo/MongoScrapbook.ts

```diff
@@ -204,4 +207,46 @@ class MongoScriptDocumentVisitor extends MongoVisitor<MongoCommand[]> {
 	protected defaultResult(_node: ParseTree): MongoCommand[] {
 		return this.commands;
 	}
+
+	private parseContext(ctx: mongoParser.ArgumentContext | mongoParser.PropertyValueContext): Object {
+		let parsedObject: Object = {};
+		if (!ctx || ctx.childCount === 0) { //Base case and malformed statements
+			return parsedObject;
+		}
+		// In a well formed expression, Argument and propertyValue tokens should have exactly one child, from their definitions in mongo.g4
+		// The only difference in types of children between PropertyValue and argument tokens is the functionCallContext that isn't handled at the moment.
+		let child: ParseTree = ctx.children[0];
```

Comment:
I assume you'll need to add support for multiple children...?

Reply:
From my understanding, 
the token has 0 children if it is an empty string. 
1 child for each TerminalNode (a comma, a dot, a bracket/brace, etc.)
1 child for each know token it matches. 

I'll keep aside the case of 0 children moving forward (That's an empty string, handled a few lines above this code). 

For [an argument](https://github.com/Microsoft/vscode-cosmosdb/blob/f3641d4a88ebe8fe06120a9d2531e9e75e2e5633/grammar/mongo.g4#L34): If matched correctly, it will have only one child, which will be a literal, ObjectLiteral, or ArrayLiteral 
For [a propertyValue](https://github.com/Microsoft/vscode-cosmosdb/blob/f3641d4a88ebe8fe06120a9d2531e9e75e2e5633/grammar/mongo.g4#L47): If matched correctly, it should have only once child, which will be literal, ObjectLiteral, ArrayLiteral  or FunctionCall. 

The respective types would have the suffix of `Context` to the identifier on the g4.